dist/oshop-dashboard/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
dist directories get generate when you build a project. They should be gitignored. add the directory to your git ignore